### ISSUE TITLE
Ensure all header casing is AWS normalized

### ISSUE THREAD

**Issue by ryanblock** [architect/architect#793]
Had additional reports of AWS header casing being inconsistent between production AWS (which has its own weird set of rules) and Express Router (which we use in Sandbox); probably need to force-lowcase everything not in/around here: https://github.com/architect/sandbox/blob/master/src/http/invoke-http/_header-fmt.js#L10-L29

**Comment by filmaj** [comment]
Using arc 6.3.5 locally, these are the values I see for the request headers (both `req.headers` as well as `req.multiValueHeaders`) for an http route in Sandbox:

```
  headers: {
    Host: 'localhost:3333',
    connection: 'keep-alive',
    'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.113 Safari/537.36',
    accept: 'image/webp,image/apng,image/*,*/*;q=0.8',
    'sec-fetch-site': 'same-origin',
    'sec-fetch-mode': 'no-cors',
    'sec-fetch-dest': 'empty',
    referer: 'http://localhost:3333/',
    'accept-encoding': 'gzip, deflate, br',
    'accept-language': 'en-US,en;q=0.9,es-MX;q=0.8,es;q=0.7,pl-PL;q=0.6,pl;q=0.5',
    Cookie: '_idx=eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4R0NNIn0..2DGOzBqLT3mZEc4l.Z3gWggwEUzJnyxWquZxNF0QsYiB79VEXW7wxhEFgkys_VVdhe6FdJqpuSyOooOw.NHgftYjhwDibDVE3bbGpRA'
  },
```

So, `User-Agent` and `Cookie`, at least, are capitalized, while the others seem not to be.

The same request but to the deployed (staging) environment has the following values for headers:

```
  headers: {
    Accept: 'image/webp,image/apng,image/*,*/*;q=0.8',
    'accept-encoding': 'gzip, deflate, br',
    'Accept-Language': 'en-US,en;q=0.9,es-MX;q=0.8,es;q=0.7,pl-PL;q=0.6,pl;q=0.5',
    'CloudFront-Forwarded-Proto': 'https',
    'CloudFront-Is-Desktop-Viewer': 'true',
    'CloudFront-Is-Mobile-Viewer': 'false',
    'CloudFront-Is-SmartTV-Viewer': 'false',
    'CloudFront-Is-Tablet-Viewer': 'false',
    'CloudFront-Viewer-Country': 'CA',
    Cookie: '_idx=eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4R0NNIn0..epD5khjyft7FSCdG.Qo6cnX8Ju6pVckjzm-TYTiLjaVsfeNwmtbxc-4bpWl9yj4kKfnBeRJMeY1F2lvA.0cB15klKIuHhGOVulBEDLA',
    Host: 'REDACTED',
    Referer: 'https://REDACTED/',
    'sec-fetch-dest': 'empty',
    'sec-fetch-mode': 'no-cors',
    'sec-fetch-site': 'same-origin',
    'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.113 Safari/537.36',
    Via: '2.0 a76a40b05a00844b502c99a4874fc37e.cloudfront.net (CloudFront)',
    'X-Amz-Cf-Id': 'PoWxp1tT75Lq33WpW2XOoay5XpybKTCvYEBZiOqOMUx1oC9IQK51Mw==',
    'X-Amzn-Trace-Id': 'Root=1-5eb16dcd-591722dc9d2f5e9451d6e1ac',
    'X-Forwarded-For': '99.230.196.156, 64.252.147.86',
    'X-Forwarded-Port': '443',
    'X-Forwarded-Proto': 'https'
  },
```

**Comment by ryanblock** [comment]
Fixed in `6.5.0`!
